modules/install/pages/install-platforms.adoc

```diff
@@ -15,9 +15,12 @@ NOTE: All listed operating systems are 64-bit.
 |===
 | *Operating System* | *Version*
 
-| Amazon Linux*
+| Amazon Linux 2* (Starting version 6.0.1)
```

Comment:
Is it still true that we only support when installed via the AMI?

Reply:
@malarky No, confirmed with Mihir that it is not the case. Removing the asterisk.